config/test/custom-matchers.ts

```diff
@@ -18,78 +18,122 @@ type PerseusRenderer = {
     guessAndScore: () => [Array<any>, PerseusScore];
 };
 
+type Answerable = PerseusRenderer | PerseusScore;
+
+function isRenderer(obj: Answerable): obj is PerseusRenderer {
+    // @ts-expect-error - TS(2339) - TS is annoying
+    return obj?.guessAndScore !== undefined;
+}
+
+function check(answerable: Answerable) {
+    let widgetState: string = "";
+    let score: PerseusScore;
+
+    if (isRenderer(answerable)) {
+        const result = answerable.guessAndScore();
+        widgetState = JSON.stringify(result[0]);
+        score = result[1];
+    } else {
+        score = answerable;
+    }
+
+    return {widgetState, score};
+}
+
 expect.extend({
-    toHaveBeenAnsweredCorrectly(renderer: PerseusRenderer) {
-        const [widgetState, score] = renderer.guessAndScore();
+    toHaveBeenAnsweredCorrectly(answerable: Answerable) {
+        const {widgetState, score} = check(answerable);
+
         if (score.type === "invalid") {
+            let errMessage = `Invalid answer: ${score.message || "(no message)"}`;
+            if (widgetState) {
+                errMessage += `; widget state ${widgetState}`;
```

Comment:
```suggestion
                errMessage += `; widget state: ${widgetState}`;
```
Totally a nit. Should this have a colon after it?